static/src/javascripts/projects/commercial/modules/remove-slots.spec.js

```diff
@@ -0,0 +1,53 @@
+import { removeSlots, removeDisabledSlots } from './remove-slots';
+
+const adSlotSelector = '.js-ad-slot';
+const toggledAdLabelSelector = '.ad-slot__label--toggle';
+
+describe('Remove ad slots and labels', () => {
+    afterEach(() => {
+        if (document.body) {
+            document.body.innerHTML = '';
+        }
+    });
+    test('Remove all ad slots and toggled labels', () => {
+        if (document.body) {
+            document.body.innerHTML = `
+                <div>
+                    <div class="ad-slot__label ad-slot__label--toggle hidden">Advertisement</div>
+                    <div class="ad-slot__label ad-slot__label--toggle hidden">Advertisement</div>
+                    <div class="ad-slot__label">Advertisement</div>
+                    <div class="js-ad-slot"></div>
+                    <div class="js-ad-slot"></div>
+                </div>
+            `;
+        }
+        expect(Array.from(document.querySelectorAll(adSlotSelector)).length).toEqual(2);
+        expect(Array.from(document.querySelectorAll(toggledAdLabelSelector)).length).toEqual(2);
```

Comment:
I think [`NodeList` has a `length` property](https://developer.mozilla.org/en-US/docs/Web/API/NodeList/length).

```suggestion
        expect(document.querySelectorAll(toggledAdLabelSelector).length).toEqual(2);
```

Reply:
Thanks. Updated in a new commit